lib/brcobranca/remessa/cnab400/itau.rb

```diff
@@ -108,12 +108,12 @@ def monta_detalhe(pagamento, sequencial)
           detalhe << pagamento.data_vencimento.strftime('%d%m%y')           # data do vencimento                    9[06]
           detalhe << pagamento.formata_valor                                # valor do documento                    9[13]
           detalhe << cod_banco                                              # codigo banco                          9[03]
-          detalhe << ''.rjust(5, ' ')                                       # agencia cobradora - deixar zero       9[05]
+          detalhe << ''.rjust(5, '0')                                       # agencia cobradora - deixar zero       9[05]
```

Comment:
Line is too long. [121/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.